tests/config.go

```diff
@@ -44,6 +44,8 @@ type Config struct {
 	InstallOperator      bool            `yaml:"install_opeartor" json:"install_opeartor"`
 	OperatorTag          string          `yaml:"operator_tag" json:"operator_tag"`
 	OperatorImage        string          `yaml:"operator_image" json:"operator_image"`
+	BackupTag            string          `yaml:"backup_tag" json:"backup_tag"`
```

Comment:
remove it? it seems not necessary.

Reply:
temporary not used, I suggest keep it, just like OperatorTag.

Reply:
`OperatorTag` is `tidb-operator` repo tag and not an image tag.

Reply:
OK, done